content/swiftui/concepts/views/terms/image/image.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'Image'
+Description: 'Allows users to incorporate images.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Image'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+An **`Image`** view is used to enhance the aesthetic of the [user interface (UI)] (https://www.codecademy.com/resources/docs/uiux) and can be functional when used correctly.
+
+## Syntax
+
+```pseudo 
+`Image`("image-example")
+    // Fitting and aspect ratios
+```
+
+A text label can be added to the `image` and the `image` can be changed to fit the requirements.
+
+## Example
+
+The following example adds the codecademy logo clipped to a circle shape:
+
+Once the `image` you would like to use has been added to the 'Assets' (Assets.xcassets) section of your correspoding project it can be called upon.
```

Comment:
```suggestion
Once the `image` to be used has been added to the 'Assets' (Assets.xcassets) section of the corresponding project, it can be called upon.
```